README.adoc

```diff
@@ -66,81 +65,51 @@ You need to have Docker installed. For installation instructions, refer to the o
-----
-
-QueryResource.java
-[source, Java, linenums, role='code_column hide_tags=copyright']
-----
-include::start/query/src/main/java/io/openliberty/guides/query/QueryResource.java[]
+include::defaultrx/query/src/main/java/io/openliberty/guides/query/client/InventoryClient.java[]
 ----
 
 Navigate to the `start` directory to begin.
 
-An implementation of the default reactive JAX-RS client is already provided for you. The classes you will change are [hotspot file=0]`InventoryClient` and [hotspot file=1]`QueryResource` which now implement the default JAX-RS reactive client.
-
-The [hotspot file=0]`InventoryClient` class is used to retrieve the inventory data. In the [hotspot file=0]`InventoryClient` class, notice how the [hotspot=getSystem file=0]`getSystem()` returns the `CompletionStage` interface. This interface represents a unit or stage of a computation, and once the associated computation completes, the value contained can be retrieved. Notice how it also contains an [hotspot=rx file=0]`rx()` call. This is the default reactive invoker for the `CompletionStage` interface and this method is what retrieves the `CompletionStageRxInvoker` class and allows for these methods to function correctly with the `CompletionStage` interface return type.
-
-The [hotspot file=1]`QueryResource` class is used to query data from the inventory service. The [hotspot=systemLoadMethod file=1]`systemLoad` endpoint processes the data, asynchronously, that is retrieved by the [hotspot=getsystem file=1]`InventoryClient`, as well as serves the data when all of the services have completed execution. The [hotspot=thenAcceptAsync file=1]`thenAcceptAsync()` and [hotspot=exceptionally file=1]`exceptionally()` methods together behave like an asynchronous try-catch block. The data is processed in the [hotspot=thenAcceptAsync file=1]`thenAcceptAsync()` method, only when the `CompletionStage` interface has finished retrieving it. It is important to note that when you return a `CompletionStage` type in the resource, it doesn’t necessarily mean that the computation completed and the response was built. A [hotspot=countdownlatch file=1]`CountDownLatch` is used to guarentee all of the requests have completed. This is something that will be reviewed further into the guide.
-
-== Updating the web client to use third-party reactive libraries
-
-JAX-RS supports the usage of third-party libraries like https://eclipse-ee4j.github.io/jersey[Jersey^]. Jersey provides a client-builder interface which allows users to make a request for the data. It allows the use of specific invokers as configuration so that the client can support https://github.com/ReactiveX/RxJava[RxJava^] objects instead of only the `CompletionStage` interface. These custom objects are useful for covering use cases that the `CompletionStage` interface cannot. 
+The CXF reactive provider is provided by default in Open Liberty so that you can write applications using reactive APIs like RxJava. 
```

Comment:
This should probably change back to the original text - CXF reactive provider is not provided by default - only the JAX-RS reactive API (which uses `CompletionStage`) is provided by default.  See previous comment.